tests/codegen/clone_as_copy.rs

```diff
@@ -1,40 +0,0 @@
-//@ revisions: DEBUGINFO NODEBUGINFO
-//@ compile-flags: -O -Cno-prepopulate-passes
```

Comment:
I think you can also add `-Zunsound-mir-opts` here.

Reply:
@DianQK is there any particular "markers" that will make this easier to find? Or is `-Zunsound-mir-opts` alone sufficient?

Reply:
> @DianQK is there any particular "markers" that will make this easier to find? Or is `-Zunsound-mir-opts` alone sufficient?

Just add it, I think: `-O -Cno-prepopulate-passes -Zunsound-mir-opts`.

Reply:
Added `-Zunsound-mir-opts` along with FIXMEs linking to original issue.